aptos-move/block-executor/src/executor.rs

```diff
@@ -1278,6 +1281,17 @@ where
                                 output.get_write_summary(),
                             )
                         });
+
+                    if !found_module_rw_conflict
+                        && !last_input_output.append_and_check_module_rw_conflict(
+                            latest_view.take_sequential_reads().module_reads.iter(),
```

Comment:
you cannot call take_sequential_reads twice - it will not be present the next time. you need to call it once into a local variable, and use it in both places here.


for testing - publish-package and  mix_publish_transfer from single_node_performance.py are good to manually see if values make sense

I think this would panic, for example in single node perf benchmark

Reply:
The tests did not panic!

However I made the change to not call 'take_sequential_reads' twice

Reply:
oh , take replaces with empty. so second call gets incorrect (always empty) value. 

you should confirm that before and after you change effective_gas for those two types of transactions changes (i.e. becomes much larger)

Reply:
you can just add label to the pr - [CICD:run-execution-performance-test](https://github.com/aptos-labs/aptos-core/labels/CICD%3Arun-execution-performance-test) (added it now)